### DISCUSSION TITLE
Total supply

### DISCUSSION THREAD

**Discussion by rhesus-protocol** [0xPolygon/polygon-edge#618] [Q&A]
Hi,
I am running polygon-edge with POS in testnetwork. My client want to put total supply of GAS Fee token.

1. How can I set total supply of the gas token?
2. Do I have to premine total supply or the validators keep mining the token?
Thanks

**Comment by laviniat1996** [comment]
Hello @rhesus-protocol ,

Thank you for your question!
The premined amount of the gas token will be the total supply.  No other amount of the native currency (gas token) can be minted later.

**Reply by rhesus-protocol** [reply]
Thank you so much for your answer.
